### ISSUE TITLE
cant schedule if argument contains equals sign

### ISSUE THREAD

**Issue by pawelmhm** [scrapinghub/shub#100]
```
shub schedule pet-supermarket.co.uk_keywords -a keywords="id1=dog"
```
fails with

```
Traceback (most recent call last):
  File "/usr/local/bin/shub", line 9, in <module>
    load_entry_point('shub==1.4.0', 'console_scripts', 'shub')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 991, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/home/pawel/scrapinghub/shub_clone/shub/schedule.py", line 18, in cli
    job_key = schedule_spider(apikey, project_id, spider, argument)
  File "/home/pawel/scrapinghub/shub_clone/shub/schedule.py", line 28, in schedule_spider
    return conn[project_id].schedule(spider, **dict(x.split('=') for x in arguments))
ValueError: dictionary update sequence element #0 has length 3; 2 is required
```

**Comment by jdemaeyer** [comment]
Thanks @pawelmhm, fixed via #101

Be aware that command line arguments/settings are always parsed as string and since don't need to be enclosed in quotes. I.e., your keywords from above are parsed as `{'keywords': '"id1=dog"'}`, and not `{'keywords': 'id1=dog'}`.

**Comment by pawelmhm** [comment]
that was quick, thanks @jdemaeyer
